components/claim/ClaimFooter.tsx

```diff
@@ -18,26 +18,31 @@ export const ClaimFooter = (props: Props) => {
   const stakeAgree = isComment && props.claim.votes().agreement !== 0 ? true : false;
   const stakeDisagree = isComment && props.claim.votes().disagreement !== 0 ? true : false;
 
+  const textAgree = props.claim.votes().agreement.toFixed(2) + " " + token;
+  const textDisagree = props.claim.votes().disagreement.toFixed(2) + " " + token;
+
   return (
     <div className="flex mt-2 px-2">
       <div className="flex-1 w-full">
         <div className="grid place-content-start w-full">
           <div className="grid place-content-start">
             {isClaim && (
               <BaseButton
+                effect={true}
                 font="font-normal"
                 icon={<TriangleUpIcon className="mb-[1px]" />}
                 position="right"
-                text={`${props.claim.votes().agreement.toFixed(2)} ${token}`}
+                text={textAgree}
               />
```

Comment:
The buttons here animate the stakes of the agreeing and disagreeing sides of the bets in the claim footer.